src/sentry/static/sentry/app/components/charts/releaseSeries.jsx

```diff
@@ -98,11 +99,19 @@ class ReleaseSeries extends React.Component {
         },
         tooltip: tooltip || {
           formatter: ({data}) => {
-            return `<div>${moment
+            const time = moment
               .tz(data.value, utc ? 'UTC' : getUserTimezone())
-              .format('MMM D, YYYY LT')} <br />
-            Release: ${formatVersion(data.name, true)}<br />
-            </div>`;
+              .format('MMM D, YYYY LT');
+            const version = escape(formatVersion(data.name, true));
+            return [
+              '<div class="tooltip-series">',
+              `<div><span class="tooltip-label"><strong>Release</strong></span> ${version}</div>`,
```

Comment:
Do we want to `t()` here? Wasn't in the old old.

Reply:
Yeah I can add that.